docs/components/alert/README.md

```diff
@@ -66,33 +66,41 @@ CdrAlert is a simple wrapper component that allows for composing various alert l
 
 There are four different options for styling the alert, based on the [alert type](../alert/#guidelines). 
 
-Alerts should be passed an icon component and text
+Alerts should be passed the icon component with the appopriate icon and text for the alert message. 
 
 <cdr-doc-example-code-pair repository-href="/src/components/alert"
 :sandbox-data="$page.frontmatter.sandboxData" >
 
 ```html
 <cdr-alert type="info">
-  <icon-information-fill /> Info Alert
+  <icon-information-fill /> Info alert
 </cdr-alert>
 <br>
 <cdr-alert type="success">
-  <icon-check-fill/> success alert
+  <icon-check-fill/> Success alert
 </cdr-alert>
 <br>
 <cdr-alert type="warning">
-  <icon-warning-fill/> warning alert
+  <icon-warning-fill/> Warning alert
 </cdr-alert>
 <br>
 <cdr-alert type="error">
-  <icon-x-fill/> error alert
+  <icon-x-fill/> Error alert
 </cdr-alert>
 ```
 </cdr-doc-example-code-pair>
 
 
 ## Accessibility
-TODO
+
+Many WCAG requirements are contextual to their implementation. To ensure that usage of this component complies with accessibility guidelines you are responsible for the following:
+- For static messaging that calls out the type of alert in the text and that is loaded with the page there is no need to do anything specific
+- For messages that do not include text to identify what type of alert is being displayed users should add screen reader only text that adds this information to the start of their message
+- Provide a text description when user input falls outside the required format or values: [W3C Techniques for WCAG 2.1](https://www.w3.org/WAI/WCAG21/Techniques/general/G83)
```

Comment:
this is specific to form validation - so maybe cut this until we cover everything that would be needed for that?

Reply:
Ok with me

Reply:
@mhewson or should we just remove these accessibility points for now? (lines 97-99)